systems/controllers/affine_controller.h

```diff
@@ -58,45 +61,49 @@ class AffineController : public drake::systems::LeafSystem<double> {
  */
 class AffineParams : public TimestampedVector<double> {
  public:
-  AffineParams(int num_states, int num_efforts):
-      TimestampedVector<double>(
-          num_states * num_efforts + num_efforts + num_states),
-      num_states_(num_states), num_efforts_(num_efforts),
-      K_(get_mutable_data().head(num_states * num_efforts).data(), num_efforts,
-         num_states),
-      E_(get_mutable_data().segment(num_states * num_efforts, num_efforts)
-                                      .data(), num_efforts),
-      desired_state_(get_mutable_data().segment(num_states * num_efforts +
-                                                num_efforts, num_states).data(),
-                     num_states) {}
-
-  int GetNumStates() const { return num_states_;}
+  AffineParams(int num_states, int num_efforts)
```

Comment:
Is this spacing really better?

Reply:
This (And the other formatting differences) is the output of running my clang formatter on the file. Not really sure what to do about it.